pkg/console/subresource/configmap/configmap.go

```diff
@@ -193,3 +193,13 @@ func authServerYaml(logoutRedirect string) yaml.MapSlice {
 func consoleBaseAddr(host string) string {
 	return util.HTTPS(host)
 }
+
+// Helper function that pulls the yaml struct out of the data section of a configmap yaml
```

Comment:
Curious if you `oc delete configmap console-config -n openshift-config-managed` does this error?

Reply:
Let me retest this.  I did something a little different by removing the data in the console-config.yaml vs not having Data at all. I will get back to you.

Reply:
Tested this again with 1) regular values, 2) cm but with no data, 3) no cm in the openshift-config-managed ns.  All passed and I did not see errors in the log.